lcls-twincat-motion/Library/POUs/Motion/FB_MotionPneumaticActuator.TcPOU

```diff
@@ -14,9 +14,9 @@ VAR_INPUT
     ibOverrideInterlock:BOOL; (*if true interlocks are ignored*)
     // Reset fault
     {attribute 'pytmc' := '
-    pv: FF_Reset
+    pv: FFO_AutoReset
     '}
-    i_xReset: BOOL;
+    i_xAutoReset: BOOL;
```

Comment:
Is it possible to include this option while also keeping the old `i_xReset` API, rather than straight replacing it? This ends up renaming a PV and renaming an argument which could be annoying for the other places where we use this FB.

Here's a search that shows all of our uses of this FB:
https://github.com/search?q=org%3Apcdshub+FB_MotionPneumaticActuator&type=code

Is isn't that many but all of these would need to be edited on update

Reply:
Good point! Thank you. I put the original API back in.